main.py

```diff
@@ -71,7 +71,16 @@ class HomePageRedirectHandler(base.BaseHandler):
     """
     def get(self):
         if self.user_id and user_services.has_fully_registered(self.user_id):
-            self.redirect(feconf.DASHBOARD_URL)
+            # Get all the explorations associated with the user's account
```

Comment:
In fact, you should probably look at the Travis checks. They're failing, and this PR needs to address that too.

You may also want to take a look at the wiki documentation for how to run tests locally.